frontend/html/favicon_io/about.txt

```diff
@@ -0,0 +1,6 @@
+This favicon was generated using the following font:
+
```

Comment:
I think you can remove this file?

Reply:
Yes, nice catch! I will make sure it's removed before merging.